MAGECLOUD-3258: warm cache with multiple pages (#4749)
magento/devdocs: guides/v2.1/cloud/env/variables-post-deploy.md

-   **multiple pages**—Use the following format to cache multiple pages according to a specific regular expression pattern:

    ```terminal
    <entity_type>:<pattern|url>:<store_id|store_code>
    ```

    -  `entity_type`: Choose `category` or `cms-page`

>     -  `pattern`: Use a `regexp` pattern or exact match `url` to filter the URLs or an asterisk (\*) for all pages 

Reviewer: ```suggestion
    -  `pattern|url`: Use a `regexp` pattern or an exact match `url` to filter the URLs, or use an asterisk (\*) for all pages
```

    -  `store_id|store_code`: Use the ID or Code of the store or an asterisk (\*) for all stores

>     The following example caches 

Reviewer: A colon introduces a list.
```suggestion
    The following example caches:
```

>     - all category pages for store with id 1

Reviewer: capitalize ID
```suggestion
    - all category pages for store with ID 1
```

    - category page `cars` for store with code `store_en`

>     - cms page `contanct` for all stores 

Reviewer: misspelling
```suggestion
    - cms page `contact` for all stores 
```

>     - any category pages that contains `car_` and end with `html` for store with id 2

Reviewer: There is a plural conflict here. Pages contain or page contains...heh heh.
```suggestion
    - any category page that contains `car_` and ends with `html` for store with ID 2
```

>     - any category pages that contains `tires_` for store with code `store_gb`

Reviewer: ```suggestion
    - any category page that contains `tires_` for store with code `store_gb`
```
